src/sdk-reference/cpp/1/kuzzle/add-listener/index.md

```diff
@@ -9,20 +9,21 @@ order: 200
 # addListener
 
 Adds a listener to an event.  
-When an event is triggered, listeners are triggered in the order in which they were added.
+When an event is triggered, listeners are triggered in the order in which they were added.  
+Theses listener will receive a `const std::string` as only argument. This string is a JSON payload representing the event.
 
 ## Signature
 
 ```cpp
-kuzzleio::KuzzleEventEmitter* addListener(kuzzleio::Event event, kuzzleio::EventListener* listener)
+kuzzleio::KuzzleEventEmitter* addListener(kuzzleio::Event event, EventListener* listener)
```

Comment:
can't we remove the namespace from the Event part also?

Reply:
The `Event` is inside the `kuzzleio` namespace.  
We have an issue to put `EventListener` inside too https://jira.kaliop.net/browse/KZL-500